cmake/dependencies/AMReX.cmake

```
@@ -294,7 +294,7 @@ set(WarpX_amrex_src ""
 set(WarpX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(WarpX_amrex_internal)")
-set(WarpX_amrex_branch "25.03"
+set(WarpX_amrex_branch "a634b09c6f227cc06952043de799942424125dcf"
```

Try https://github.com/AMReX-Codes/amrex/commit/107db66f8723799cc0a25e45759c4d28a1dfeffe to fetch latest commit before the merge of https://github.com/AMReX-Codes/amrex/pull/4357:
```suggestion
set(WarpX_amrex_branch "107db66f8723799cc0a25e45759c4d28a1dfeffe"
```